ui/src/admin/actions/chronograf.js

```diff
@@ -10,6 +10,10 @@ import {
   createOrganization as createOrganizationAJAX,
   updateOrganization as updateOrganizationAJAX,
   deleteOrganization as deleteOrganizationAJAX,
+  getMappings as getMappingsAJAX,
+  createMapping as createMappingAJAX,
+  updateMapping as updateMappingAJAX,
```

Comment:
It is really confusing that these names are so similar to the action creators on 108 for example.

Reply:
I agree the naming is confusing. We should revisit this soon.